### PR TITLE
Bootstrap

### PR DESCRIPTION
**Pull request by dsandoval11**
## Summary
- Kickoff angularjs

## Screenshot
image

## Trello Card
https://trello.com/c/UrS1oyAJ/22-wolox-front-end-bootstrap

### CODE REVIEW

File: src/components/navbar/navbar.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<nav class="navbar-container">
+  <div class="navbar">
+    <img class="logo" ng-src="{{$ctrl.logoWolox}}" alt="Logo" />
+    <span class="logout">Logout</span>
+  </div>
+</nav>
```
Comment (mcallegari10) [inline src/components/navbar/navbar.html:6]: Add EOF line

File: src/components/navbar/navbar.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<nav class="navbar-container">
+  <div class="navbar">
+    <img class="logo" ng-src="{{$ctrl.logoWolox}}" alt="Logo" />
```
Comment (mcallegari10) [inline src/components/navbar/navbar.html:3]:
```suggestion
    <img class="logo" ng-src="{{$ctrl.logoWolox}}" alt="wolox logo" />
```

**Review by mcallegari10 — APPROVED** [review]

File: src/components/navbar/navbar.html (hunk 1 above)
Comment (dsandoval11) [reply src/components/navbar/navbar.html:6]: Fixed in PR #14

File: src/components/navbar/navbar.html (hunk 2 above)
Comment (dsandoval11) [reply src/components/navbar/navbar.html:3]: Fixed in PR #14

**Pull request merged**